Source/VolunteerReporting/Web/Web.csproj

```diff
@@ -10,6 +10,7 @@
   <ItemGroup>
     <PackageReference Include="DocumentFormat.OpenXml" Version="2.8.1" />
     <PackageReference Include="FakeItEasy" Version="4.1.0" />
+    <PackageReference Include="iTextSharp" Version="5.5.13" />
```

Comment:
I think the license for ITextSharp is fine, someone with more insight into that kind of stuff needs to look at that.